### PR TITLE
Enable event tracing for Tizen.

### PR DESCRIPTION
**Pull request by Dmitri-Botcharnikov**
This PR enables generation of LTTng event providers for Tizen platform by default.

### CODE REVIEW

**Comment by BruceForstall** [comment]
```
07:06:17 CMake Error at src/pal/src/configure.cmake:1346 (message):
07:06:17   Cannot find liblttng-ust-dev.  Try installing liblttng-ust-dev (or the
07:06:17   appropriate packages for your platform)
07:06:17 Call Stack (most recent call first):
07:06:17   src/pal/src/CMakeLists.txt:5 (include)
```

**Comment by BruceForstall** [comment]
cc @parjong @janvorli 

Who is the best to review this?

**Comment by janvorli** [comment]
@BruceForstall, @Dmitri-Botcharnikov the change looks good, but it looks like we don't have the lttng-ust-dev package installed in the Tizen rootfs we are using and we need to add it. 
Last time the rootfs was updated by @hqueue in https://github.com/dotnet/coreclr/commit/a23cd5287f9be53f47c78cf99dfc114518a506d0.

**Comment by hseok-oh** [comment]
As @janvorli said, we need to 1) fix rootfs script 2) update docker image for ci because lttng-ust-dev is not installed.

File: clrfeatures.cmake [hunk 1]
```diff
@@ -1,5 +1,5 @@
 if(CLR_CMAKE_TARGET_TIZEN_LINUX)
-  set(FEATURE_EVENT_TRACE 0)
+  set(FEATURE_EVENT_TRACE 1)
```
Comment (hseok-oh) [inline clrfeatures.cmake]: If you want to enable this feature, you can simply remove this line. 
`FEATURE_EVENT_TRACE` is set by default on Linux/ARM32.

**Comment by hqueue** [comment]
@Dmitri-Botcharnikov @BruceForstall  @hseok-oh  I'm now testing with new rootfs with lttng and will update  docker image used for Tizen CI if there is no problem.

File: clrfeatures.cmake (hunk 1 above)
Comment (Dmitri-Botcharnikov) [reply clrfeatures.cmake]: @hseok-oh Thank you for suggestion, but I think it's more reliable  to explicitly set feature on.

**Comment by Dmitri-Botcharnikov** [comment]
@hqueue Thank you! IMHO we need to add lttng-ust-dev package to tizen-fetch.sh script.

**Comment by hqueue** [comment]
I've just opened #14173 and #14173 should be merged before this PR.
We have to wait CI testing for #14173 to see a new docker image working correctly.

**Comment by hqueue** [comment]
@Dmitri-Botcharnikov #14173 is merged now. I think Tizen CI will work without problem after rebasing this PR to the latest master.

**Comment by Dmitri-Botcharnikov** [comment]
@dotnet-bot test Ubuntu16.04 armlb Cross Debug Build please

**Comment by Dmitri-Botcharnikov** [comment]
@dotnet-bot test Tizen armel Cross Release Build please

**Comment by Dmitri-Botcharnikov** [comment]
@hqueue Thank you!

**Comment by BruceForstall** [comment]
@janvorli Tests have passed; is this good to merge?

**Pull request merged**